applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaTypes.cpp

```diff
@@ -1,53 +0,0 @@
-/******************************************************************************
-*                 SOFA, Simulation Open-Framework Architecture                *
-*                    (c) 2006 INRIA, USTL, UJF, CNRS, MGH                     *
-*                                                                             *
-* This program is free software; you can redistribute it and/or modify it     *
-* under the terms of the GNU Lesser General Public License as published by    *
-* the Free Software Foundation; either version 2.1 of the License, or (at     *
-* your option) any later version.                                             *
-*                                                                             *
-* This program is distributed in the hope that it will be useful, but WITHOUT *
-* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
-* FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License *
-* for more details.                                                           *
-*                                                                             *
-* You should have received a copy of the GNU Lesser General Public License    *
-* along with this program. If not, see <http://www.gnu.org/licenses/>.        *
-*******************************************************************************
-* Authors: The SOFA Team and external contributors (see Authors.txt)          *
-*                                                                             *
-* Contact information: contact@sofa-framework.org                             *
-******************************************************************************/
-#include <sofa/gpu/cuda/CudaTypes.h>
-
-#include <sofa/core/ObjectFactory.h>
-
-#include <sofa/core/MultiMapping.inl>
-
-namespace sofa
-{
-
-namespace core
-{
-
-using namespace sofa::defaulttype;
-using namespace sofa::gpu::cuda;
-
-
-template class SOFA_GPU_CUDA_API MultiMapping<CudaVec3Types, Vec1Types>;
```

Comment:
Here it is instanciations of MM[CudaTypes,NonCudaTypes] and in https://github.com/sofa-framework/sofa/blob/173dd8711db6eb3cca6014e01cf046ef2cbfcb78/applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaMultiMapping.cpp#L29
I can see only instanciations of MM[CudaTypes,CudaTypes],
Should we not keep it ? 🤔
or at least move those to CudaMultiMapping ? (it would make more sense actually)

Reply:
Bien vu!